manifests/plugin.pp

```diff
@@ -1,11 +1,12 @@
 #
 define collectd::plugin (
-  $ensure   = 'present',
-  $content  = undef,
-  $order    = '10',
-  $globals  = false,
-  $interval = undef,
-  $plugin   = $name
+  $ensure        = 'present',
+  $content       = undef,
+  $order         = '10',
+  $globals       = false,
+  $interval      = undef,
+  $flushinterval = undef,
```

Comment:
can you please add a datatype to the new parameter?

Reply:
done !